src/Symfony/Component/Security/Http/RememberMe/AbstractRememberMeServices.php

```diff
@@ -285,7 +285,18 @@ protected function cancelCookie(Request $request)
             $this->logger->debug(sprintf('Clearing remember-me cookie "%s"', $this->options['name']));
         }
 
-        $request->attributes->set(self::COOKIE_ATTR_NAME, new Cookie($this->options['name'], null, 1, $this->options['path'], $this->options['domain']));
+        // Hard coded the default values for secure and http only, would be better if those where constants
+        $request->attributes->set(
+            self::COOKIE_ATTR_NAME,
+            new Cookie($this->options['name'],
+                null,
+                1,
+                $this->options['path'],
+                $this->options['domain'],
+                isset($this->options['secure']) ? $this->options['secure'] : false,
```

Comment:
It is common practice

https://github.com/symfony/symfony/blob/6393ec31690a3ecc73e5f1f7ea2185cda7aba203/src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php#L66
https://github.com/symfony/symfony/blob/2aeaa22e3171b5e1f10f2cc2a7ff7f6608b65f9d/src/Symfony/Component/Security/Http/Authentication/DefaultAuthenticationFailureHandler.php#L75
https://github.com/symfony/symfony/blob/aad7963ede0db55cb85290912018e4bea6b0ba7a/src/Symfony/Component/HttpFoundation/Session/Storage/Handler/MongoDbSessionHandler.php#L82